spec.md

```diff
@@ -211,15 +213,17 @@ extmodule MyExternalModule :
   parameter y = 42
 ```
 
-An example of an externally defined module with references is:
+An example of an externally defined module with references and groups is:
 
 ```firrtl
-extmodule MyExternalModuleWithRefs :
+extmodule MyExternalModuleWithRefsAndGroups :
   input foo : UInt<2>
   output mysignal : Probe<UInt<1>>
   output myreg : RWProbe<UInt<8>>
   ref mysignal is "a.b"
   ref myreg is "x.y"
+  defgroup MyGroup
+  defgroup MyOtherGroup
```

Comment:
can you show what this looks like for a nested group?

Reply:
Will add this in a later commit.